.github/workflows/docker.yml

```diff
@@ -0,0 +1,44 @@
+name: Create and publish Docker image
+
+on:
+  push:
+    branches: ['main']
```

Comment:
We might need to have a way to test building (but not releasing) the container for commits, so people know nothing breaks on the container side whenever they propose a change.

Reply:
I'll need to investigate on that part